libraries/azurerm_resource.rb

```diff
@@ -5,26 +5,39 @@
 class AzurermResource < Inspec.resource(1)
   supports platform: 'azure'
 
+  MGMT_HOST = 'https://management.azure.com'
+  GRAPH_HOST = 'https://graph.windows.net'
+
   def exists?
     @exists ||= false
   end
 
   def client
     Azure::Management.instance
-                     .with_client(azure_client)
+                     .with_client(rest_client(MGMT_HOST))
```

Comment:
I think leaving a default for `MGMT_HOST` is a reasonable default. I'd also recommend spelling out `MANAGEMENT_HOST`.